acceptance/bundle/deploy/jobs/fail-on-active-runs/databricks.yml.tmpl

```diff
@@ -0,0 +1,15 @@
+bundle:
+  name: deploy-jobs-active-runs-test-$UNIQUE_NAME
+
+resources:
+  jobs:
+    foo:
+      name: test-job-basic-{{.unique_id}}
```

Comment:
`{{.unique_id}}` this looks like a left over from previous impl? If the test passes, then it does not need to be unique and can be simply removed.

Reply:
Replaced with $UNIQUE_NAME in the follow-up commit. I think leaving the name static has a risk of having side effects